<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Расходы на сегодня</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableOutlays"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Доход за сегодня</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableRent"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Доходрасход за срок</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
              <item>
               <widget class="QCalendarWidget" name="calendarFirst">
                <property name="styleSheet">
                 <string notr="true">QCalendarWidget QAbstractItemView:enabled 
  {
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(3, 255, 255); 
  }</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCalendarWidget" name="calendarSecond">
                <property name="styleSheet">
                 <string notr="true">QCalendarWidget QAbstractItemView:enabled 
  {
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(3, 255, 255); 
  }</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="tableCash">
              <property name="styleSheet">
               <string notr="true">QCalendarWidget QToolButton {
  	height: 60px;
  	width: 150px;
  	color: white;
  	font-size: 24px;
  	icon-size: 56px, 56px;
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333);
  }
  QCalendarWidget QMenu {
  	width: 150px;
  	left: 20px;
  	color: white;
  	font-size: 18px;
  	background-color: rgb(100, 100, 100);
  }
  QCalendarWidget QSpinBox { 
  	width: 150px; 
  	font-size:24px; 
  	color: white; 
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }
  QCalendarWidget QSpinBox::up-button { subcontrol-origin: border;  subcontrol-position: top right;  width:65px; }
  QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right;  width:65px;}
  QCalendarWidget QSpinBox::up-arrow { width:56px;  height:56px; }
  QCalendarWidget QSpinBox::down-arrow { width:56px;  height:56px; }
   
  /* header row */
  QCalendarWidget QWidget { alternate-background-color: rgb(128, 128, 128); }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:24px;  
  	color: rgb(180, 180, 180);  
  	background-color: black;  
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(0, 255, 0); 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(64, 64, 64); 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Кол-во дней до тех. обслуживания</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableTech"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>12</y>
          <width>766</width>
          <height>320</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Введите Ф.И.О</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineFio"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Введите дату заказа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCalendarWidget" name="calendar_rentCar"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Выберите машину</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxCars"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_rentCar">
             <property name="text">
              <string>Заказать</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Номер авто:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_id"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Цвет авто:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_colour"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Модель:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_model"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
             <property name="spacing">
              <number>7</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Аренда/день:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_rent"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_addcar">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Лицензия:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_license"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Тех. обсл.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_t_o"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Замена колес:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_wheels"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,3">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Страховка:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_insurance"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_change">
             <property name="text">
              <string>Изменить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
